python/eos/data/native.py

```diff
@@ -195,13 +202,22 @@ def density(self):
 
 
     @staticmethod
-    def create(path, density):
+    def create(path, density, varied_parameters=None, sigma_test_stat=None, samples=None, weights=None):
```

Comment:
I created a new issue (#848) to remind us to make ``varied_parameters`` mandatory for creating any datafile.
We should merge this as is though.